iroha/src/maintenance.rs

```diff
@@ -0,0 +1,15 @@
+//! `Maintenance` module provides structures and implementation blocks related to `Iroha`
+//! maintenance functions like Healthcheck, Monitoring, etc.
+
+use iroha_derive::Io;
+use parity_scale_codec::{Decode, Encode};
+
+/// `Health` enumerates different variants of Iroha `Peer` states.
+/// Each variant can provide additional information if needed.
+#[derive(Clone, Debug, Io, Encode, Decode)]
+pub enum Health {
+    /// `Healthy` variant means that `Peer` has finished initial setup.
+    Healthy,
+    /// `Ready` variant means that `Peer` bootstrapping completed.
+    Ready,
```

Comment:
Why not `Bootstrapped`? I think `Ready` may be confusing in the sense of healthiness

Reply:
As described in RFC https://wiki.hyperledger.org/display/iroha/Maintenance+Endpoint we may take inspiration from Prometheus cause we are going to use it.

Reply:
It is still a bit confusing, in general when our service is Ready it is also Healthy, and this fact is reflected in Prometheus API as they have these endpoints separated.